models/topics.models.js

```diff
@@ -1,8 +1,17 @@
 const db = require('../db/connection.js');
+const fs = require('fs/promises');
+
 
 exports.selectTopics = () => {
   return db.query(`SELECT * FROM topics`)
   .then(({rows}) => {
     return rows
   })
-}
\ No newline at end of file
+}
+
+exports.selectAllEndPoints = () => {
```

Comment:
You don't need to read a json, to a string, parse it back to a json and send it to the controller. This can all be handled in the controller with the json required in.

Json files required in behave like js objects, what are we wanting to send the client?